### ISSUE TITLE
FodyWeavers not being properly renamed

### ISSUE THREAD

**Issue by dansiegel** [dansiegel/Prism-Templates#3]
When creating a new project from the template the correct Weavers file is being excluded via the wildcard, however the rename is not occurring.  This problem exists with both `"**/RealmFodyWeavers.xml": "FodyWeavers.xml"` and specifying like `"src/MobileApp/RealmFodyWeavers": "src/MobileApp/FodyWeavers.xml"`

@mlorbetske not sure if perhaps you see what the problem would be here?

```json
  "sources": [
    {
      "modifiers": [
...
        {
          "condition": "(UseRealm)",
          "rename": {
            "**/RealmFodyWeavers.xml": "**/FodyWeavers.xml"
          },
          "exclude": [
            "**/NormalFodyWeavers.xml"
          ]
        },
        {
          "condition": "(!UseRealm)",
          "rename": {
            "**/NormalFodyWeavers.xml": "**/FodyWeavers.xml"
          },
          "exclude": [
            "**/RealmFodyWeavers.xml"
          ]
        },
....
      ]
    }
  ],
```

**Comment by seancpeters** [comment]
@dansiegel Mike asked me to look into this issue. We don't currently support globs in renames, but this led me to discover a bug the rename code, which is getting fixed with https://github.com/dotnet/templating/pull/1040

Once that fix is in place you'll be able to use renames without any path info as if they were globs, so if you setup a rename like:
```
"rename": {
            "RealmFodyWeavers.xml": "FodyWeavers.xml"
          },
```
... it will rename all files in your template named RealmFodyWeavers.xml - regardless of their directory.

And if a rename is setup with a relative path in the source, it will only rename that instance.

**Comment by dansiegel** [comment]
@seancpeters perfect... thanks for looking into that... it's been driving me crazy trying to figure it out.

**Comment by dansiegel** [comment]
This has been confirmed as fixed using the dotnet cli 2.0.0-preview3-006756
